portal.go

```diff
@@ -57,6 +57,11 @@ func AddRoute(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+    if !authProvider.CheckDestinationIP(internalip, username) {
+        w.Write([]byte("ACL ERROR"))
```

Comment:
The format is `ERROR {reason}` it will be relevant, in future when doing error handling